src/bot/index.ts

```diff
@@ -114,8 +121,10 @@ bot.on(mSDK.RoomEvent.Timeline, (event: mSDK.MatrixEvent) => {
       logger.error("⭕ An error occurred when checking the balance", e);
     });
   } else if (action === "!drip") {
+    sendReaction(roomId, eventId, "👀");
     if (!arg0) {
-      logger.warn("Address not provided, skipping");
+      sendReaction(roomId, eventId, "😕");
+      sendMessage(roomId, `${sender} Please provide an address to be funded.`);
       return;
```

Comment:
Sounds like there's no need to send 👀 reaction, if we're sending a 😕 right after.  
Why not move `sendReaction(roomId, eventId, "👀");` after synchronous checks?

Reply:
I see what you mean, also thought about it.

In my pedantic mind it's more consistent to always have the same pattern:

- `seen 👀` -> `👍 OK`
- `seen 👀` -> `😕 FAIL`

Changing this would introduce a `😕 FAIL` for a message that "has not been seen" in contrast with others.

Those were just my thoughts, don't know if it makes sense :D